docs/assemblies/tls.adoc

```diff
@@ -131,19 +131,22 @@ When tlsEnabled is set to True on the nodeset, and tlsCert is defined for the da
 service, certificates will be requested from the certmanager issuer designated in the issuer attribute
 (or a default) as described above.
 
-The contents of the certificate (subject name, subject alternative names, etc.) is defined using the
+The contents of the certificate (subject name, subject alternative names, etc.) are defined using the
 contents and issuer attributes as described above.
 
 The certficates are generated when an OpenstackDataplaneDeployment is created, but before any ansible EE
 pods are created.
 
 When the certificates are created, certmanager stores the certificates in secrets which are named
-"cert-<service_name>-<node_name>".  Each secret contains the certificate, key and cacert.
+"cert-<service_name>-<node_name>-#".  The # symbol represents the secret number, beginning with 0.
+As Kubernetes distributions such as Red Hat Openshift have a maximum size for a secret (currently around
+1 MiB), should the number of certificates and keys to be created is big enough to fit more than one secret,
+multiple secrets are created.  Each secret receives its number and contains the certificate, key and cacert.
 
-The certificates for all the nodes in the nodeset for a given service are all collected in a secret named
-"<nodeset>-<service_name>-certs".  This secret is the one that is mounted in the ansibleEE when
-when addCertMounts is enabled.  The code here is also slated to be refactored soon to ensure that we are
-able to accomodate large deployments.
+The certificates for all the nodes in the nodeset for a given service are all collected in secrets named
+"<nodeset>-<service_name>-certs-#".  The # symbol has the same logic of the one explained on the above
+paragraph.  These secrets are mounted in the ansibleEE when addCertMounts is enabled.  The code here is
+also slated to be refactored soon to ensure that we are able to accomodate large deployments.
```

Comment:
```suggestion
The certificates for all the nodes in the node set for a given service are collected in secrets named "<nodeset>-<service_name>-certs-#", where the `#` symbol represents the generated secret number that starts at `0`. These secrets are mounted in the ansibleEE when `addCertMounts` is enabled. 
```
Suggestion includes:

- Removing second "all" in first sentence.
- Representing "node sets" in text as two words to align with the published DP3 doc. `NodeSet`, all one word, is only used for the `OpenStackDataPlaneNodeSet` CR.
- Replacing sentence that includes "on the above paragraph" - we try to avoid referring to other parts of the doc that are "above" or "below". We can use links, but otherwise avoid placement statements. 
- Removing "The code here is also slated to be refactored soon to ensure that we are able to accommodate large deployments" cos it talks about something that hasn't happened yet - we generally avoid statements about future wishes in the docs. If this statement is important to include, then maybe try something like: "The ability to accommodate large deployments will be available in a later release."

Reply:
Ok, will adjust the text as suggested.